src/components/Blog/Post/Markdown/index.tsx

```diff
@@ -1,17 +1,39 @@
 import React from 'react'
 import rehypeReact from 'rehype-react'
+import cn from 'classnames'
 
 import * as styles from './styles.module.css'
 
 interface IMarkdownProps {
   htmlAst: Node
 }
 
+const Admonition: React.FC<{
```

Comment:
For now, I've placed the admonition element in the blog for reviewing. Once the design and code is approved, I'll add it to the docs as well. Speaking of which, should both the blog and docs use the same component taking from `gatsby-theme-iterative-docs` or do we want them to stay separate?

Reply:
I'd say put it in the theme and share it since our other sites' docs engines will want to use this component.

Reply:
Moved component to `gatsby-theme-iterative-docs`!